rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -953,6 +953,10 @@ public Optional<Keccak256> getPegoutCreationRskTxHashByBtcTxHash(Sha256Hash btcT
             return Optional.empty();
         }
 
+        if (btcTxHash == null){
+            return Optional.empty();
+        }
+
```

Comment:
```suggestion
        if (!activations.isActive(RSKIP298) || btcTxHash == null){
            return Optional.empty();
        }
```
Bit shorter. Also, we should add a test for this case